MAUI/Calendar/events.md

```diff
@@ -0,0 +1,182 @@
+    * [OldValue] : Returns the old selected date.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      SelectionChanged="OnCalendarSelectionChanged">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+this.Calendar.SelectionChanged += this.OnCalendarSelectionChanged;
+
+private void OnCalendarSelectionChanged(object sender,  private void OnCalendarSelectionChanged(object sender, CalendarSelectionChangedEventArgs e)
+    {
+        var oldDateTime = e.OldValue;
+        var newDateTime = e.NewValue;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+The `Tapped` event is triggered first, followed by 'selection-changed' event will be performed.
+
+## Tapped
+
+A [Tapped] event is triggered, each time a calendar view is tapped.
+Below is a list of the arguments:
+
+* `Sender`: This contains the `SfCalendar` object.
+
+* `Tapped`: The tapped action performed on an Calendar element can be found in the [CalendarTappedEventArgs], you can see details about the tapped dates and elements.
+
+    * [Date] : Returns the selected date.
+    * [Element] : Returns the `Calendar` element tapped.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      Tapped="OnCalendarTapped">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+this.Calendar.Tapped += OnCalendarTapped;
+private void OnCalendarTapped(object sender, CalendarTappedEventArgs e)
+{
+    var selectedDate = e.Date;
+    var calendarElement = e.Element;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## DoubleTapped
+
+Whenever the [SfCalendar] elements are double-tapped onto the view, the [DoubleTapped] event occurs. Below is a list of the arguments:
+
+* `Sender`: This contains the `SfCalendar` object.
+
+* `DoubleTapped`: This double tapped action performed on a Calendar element can be found in the [CalendarDoubleTappedEventArgs], and you can see the details about the double-tapped dates and elements.
+
+    * [Date] : Returns the double-tapped date.
+    * [Element] : Returns the double-tapped `Calendar` element.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      DoubleTapped="OnCalendarDoubleTapped">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+this.Calendar.DoubleTapped += this.OnCalendarDoubleTapped;
+
+private void OnCalendarDoubleTapped(object sender, CalendarDoubleTappedEventArgs e)
+{
+    var selectedDate = e.Date;
+    var calendarElement = e.Element;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## LongPressed
+
+Whenever the [SfCalendar] elements are long-pressed onto the view, the [LongPressed] event is triggered. Below is a list of the arguments:
```

Comment:
The following is a list of the arguments:

Reply:
Content changed